src/app/MessageDef/StatusElement.h

```diff
@@ -45,6 +45,13 @@ enum
     kCsTag_ClusterId    = 4
 };
 
+struct Type
```

Comment:
I don't know what the 'Element' suffix means, but I see it is ubiquitous in 'src/app/MessageDef'.  But by the time we get to this struct, we have: `chip::app::StatusElement::Type`.  That's a bit confusing when what this actually is is a StatusReport, isn't it?  Can we just call it that?